packages/ndla-ui/src/BannerCard/BannerCard.tsx

```diff
@@ -84,7 +83,9 @@ export const BannerCard = ({ link, title, content, linkText, image }: BannerProp
       <TextWrapper>
         <TitleText>{title}</TitleText>
         <ContentText>{content}</ContentText>
-        <LinkText href={link}>{linkText}</LinkText>
+        <LinkText target="_blank" to={link}>
```

Comment:
Trur ikkje du skal ha target blank for då får du ny fane. Om du må ha target så bruk heller `_self`.

Reply:
Sant, denne skal vel ta brukeren direkte dit.